goto/gotomagic/git.py

```diff
@@ -3,11 +3,7 @@
 import sys
 import git
 from .magic import GotoMagic
-
-
-# HACK HACK
-GOTOPATH = os.environ.get('GOTOPATH', os.path.expanduser('~/.goto'))
-
+from . import utils
 
 
 class GitMagic():
```

Comment:
This class is here still, but currently unused. Soon to be implemented properly ^^